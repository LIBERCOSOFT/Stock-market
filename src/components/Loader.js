import React from 'react';
import PropagateLoader from 'react-spinners/PropagateLoader';

const Loader = () => (
  <div className="loader__container">
    <PropagateLoader color="#070b12" />
  </div>
);

export default Loader;
