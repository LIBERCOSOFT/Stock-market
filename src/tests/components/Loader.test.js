import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Loader from '../../components/Loader';

test('renders loader component snapshot', () => {
  const { asFragment } = render(
    <Loader />,
  );
  expect(asFragment()).toMatchSnapshot();
});
