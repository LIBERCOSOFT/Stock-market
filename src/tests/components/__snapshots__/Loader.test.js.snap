// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader component snapshot 1`] = `
<DocumentFragment>
  <div
    class="loader__container"
  >
    <span
      style="display: inherit; position: relative;"
    >
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-0 1.5s infinite; animation-fill-mode: forwards;"
      />
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-1 1.5s infinite; animation-fill-mode: forwards;"
      />
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-2 1.5s infinite; animation-fill-mode: forwards;"
      />
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-3 1.5s infinite; animation-fill-mode: forwards;"
      />
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-4 1.5s infinite; animation-fill-mode: forwards;"
      />
      <span
        style="position: absolute; font-size: 5px; width: 15px; height: 15px; background: rgb(7, 11, 18); border-radius: 50%; animation: react-spinners-PropagateLoader-propogate-5 1.5s infinite; animation-fill-mode: forwards;"
      />
    </span>
  </div>
</DocumentFragment>
`;
